MeshPlugin/MeshClass.cs

```diff
@@ -0,0 +1,1580 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using AssetsTools;
+using AssetsTools.NET;
+using System.Drawing.Printing;
+using SevenZip.Compression.LZMA;
+using System.Linq;
+using System;
+using AssetsTools.NET.Extra;
+using System.Diagnostics;
+using System.IO;
+using SkiaSharp;
+using System.Resources;
+using Avalonia.Media;
+using Avalonia.Controls;
+
+namespace MeshPlugin
+{
+    public class MeshClass
```

Comment:
Please put these in separate classes. Even if it was like that in AssetStudio, I'd rather not have 30 classes in a single file.